### PR TITLE
Added RemoveVowel and tests for issue 20.

### PR DESCRIPTION
**Pull request by ZakBibi**

### CODE REVIEW

File: CodingPractice/PythonAssignments/tests/test_RemoveVowels.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class RemoveVowelTest(unittest.TestCase):
+
+    def test_00_remove_vowels_from_word_starting_with_consonant(self):
+        word = 'Constantinople'
+        self.assertEqual('Cnstntnpl', remove_vowels(word))
+
+    def test_01_remove_vowels_from_word_starting_with_vowel(self):
+        word = 'abacus'
+        self.assertEqual('abcs', remove_vowels(word))
+
```
Comment (BasilBibi) [inline CodingPractice/PythonAssignments/tests/test_RemoveVowels.py:14]: Any more tests you can think of ?

File: CodingPractice/PythonAssignments/excercises/RemoveVowels.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+def remove_vowels(word):
+    vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
+    abbreviation = []
+    abbreviation.append(word[0])
+    for w in word[1:]:
```
Comment (BasilBibi) [inline CodingPractice/PythonAssignments/excercises/RemoveVowels.py:6]: Is there a way to turn this for loop into a list comprehension ?

Comment (ZakBibi) [reply CodingPractice/PythonAssignments/excercises/RemoveVowels.py:6]: I did try this as a list comprehension, and it looks like this:

``` 
[abbreviation.append(w)
     for w in word[1:]
     if w not in vowels]
```
I do not think it makes it any easier to read.

Comment (BasilBibi) [reply CodingPractice/PythonAssignments/excercises/RemoveVowels.py:6]: Agreed.

**Review by BasilBibi — APPROVED** [review]
Ok this is ready to be merged to develop good work.

**Pull request merged**